apps/api-gateway/src/public-data/public-data.controller.spec.ts

```diff
@@ -0,0 +1,82 @@
+import { NotFoundException } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { PostCategory } from 'apps/public-data/src/public-data/entities/post-category.entity';
+import { firstValueFrom, of } from 'rxjs';
+
+import { PublicDataController } from './public-data.controller';
+import { PublicDataService } from './public-data.service';
+
+describe('PublicDataController', () => {
+  let controller: PublicDataController;
+  let service: PublicDataService;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      controllers: [PublicDataController],
+      providers: [
+        {
+          provide: PublicDataService,
+          useValue: {
+            getPostData: jest.fn(),
+            getOnePostData: jest.fn(),
+          },
+        },
+      ],
+    }).compile();
+    controller = module.get<PublicDataController>(PublicDataController);
+    service = module.get<PublicDataService>(PublicDataService);
+  });
+
+  it('api-gateway/public-data controller should be defined', () => {
+    expect(controller).toBeDefined();
+    expect(service).toBeDefined();
+  });
+
+  describe('postData', () => {
+    it('전체 post-category Data 조회', async () => {
+      const mockPostCategories = [{ id: 1, post_category_name: '팀원 모집' }];
```

Comment:
이렇게 작성해 주는 게 좋을 것 같습니다.
```suggestion
      const mockPostCategories:PostCategory[] = [{ id: 1, post_category_name: '팀원 모집' }];
```